app/models/maintenance_tasks/runner.rb

```diff
@@ -37,14 +37,17 @@ def initialize(run)
     #   for the Task to iterate over when running, in the form of an attachable
     #   (see https://edgeapi.rubyonrails.org/classes/ActiveStorage/Attached/One.html#method-i-attach).
     #   Value is nil if the Task does not use CSV iteration.
+    # @param params [Hash] the parameters to make accessible to the Task and to
+    #   persist to the Run.
```

Comment:
Maybe we could put that in chronological order, first the arguments are persisted to the run, and then they're made accessible to the Task?